build/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.15 as build
+FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.20-openshift-4.16 as build
```

Comment:
We are missing these image updates in build/Dockerfile.ci and rest of the other files.